.travis.yml

```diff
@@ -25,6 +25,6 @@ script:
     # Setup Avocado-vt for functional tests
     - AVOCADO_LOG_DEBUG=yes avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
     # Run tests
-    - inspekt checkall --disable-lint W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003,W605 --disable-style W605,W606,E501,E265,W601,E402,E722,E741 --exclude avocado-libs,scripts/github --no-license-check
+    - inspekt checkall --disable-lint W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003,W605 --disable-style W605,W606,E501,E265,W601,E402,E722,E741 --exclude avocado-libs,scripts/github,virttest/utils_net.py --no-license-check
```

Comment:
excluding the file virttest/utils_net.py directly is a little risky in my opinion. Is there any reason for it?

Reply:
utils_net.py itself raise warning in previous travis.org, environment but this time it leads to direct failure in travis.com,
see details from https://app.travis-ci.com/github/avocado-framework/avocado-vt/jobs/521743727

Reply:
It is not suitable to disable the utils file of avocado-vt in trivals-ci 
I saw there is 1 new error  E1133:2555,19: VirtIface.__getstate__: Non-iterable value self.__class__.__all_slots__ is used in an iterating context 
I think you can try to resolve it or some other methods instead of disable it.

Reply:
Yes, I agree. It should be ok if you disable E1113 in-line:
```
+        # pylint: disable=E1133
         for key in self.__class__.__all_slots__:
```
```

$ vt-check.sh utils_net.py 
PEP8 disabled: W605,W606,E501,E265,W601,E402,E722,E741
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003,W605
Pylint enabled : W0611
License check: disabled
************* Module avocado-vt.virttest.utils_net
I1101:3457,14: get_ip_address_by_interface: Module 'netifaces' has no 'AF_INET6' member, but source is unavailable. Consider adding this module to extension-pkg-whitelist if you want to perform analysis based on run-time introspection of living objects.
I1101:3460,14: get_ip_address_by_interface: Module 'netifaces' has no 'AF_INET' member, but source is unavailable. Consider adding this module to extension-pkg-whitelist if you want to perform analysis based on run-time introspection of living objects.
I1101:3462,11: get_ip_address_by_interface: Module 'netifaces' has no 'ifaddresses' member, but source is unavailable. Consider adding this module to extension-pkg-whitelist if you want to perform analysis based on run-time introspection of living objects.
(p3-env) [yicui@yicui virttest]$ echo $?
0
```